src/Components/test/testassets/BasicTestApp/BindCasesComponent.razor

```diff
@@ -32,7 +32,7 @@
 </p>
 <p>
     Nullable int:
-    <input id="textbox-nullable-int" @bind="textboxNullableIntValue" type="number" />
+    <input id="textbox-nullable-int" @bind-value="textboxNullableIntValue" type="number" />
```

Comment:
I think this file (and `GlobalizationBindCases.razor`) could use a comment explaining that the mixture of `@bind` and `@bind-value` is arbitrary, both should work the same, and we're mixing them just for extra code coverage.

Otherwise in the future we'll be struggling to work out why certain cases need `-value` and others don't.

Alternatively, we could have a couple of dedicated `-value` tests independently of the existing set.

Reply:
Added commentz.